src/mmw/js/src/data_catalog/views.js

```diff
@@ -849,10 +852,77 @@ var CuahsiSwitcherView = Marionette.ItemView.extend({
     }
 });
 
+var ResultDetailsUSGSWQPView = ResultDetailsBaseView.extend({
+    template: resultDetailsUSGSWQPTmpl,
+
+    templateHelpers: function() {
+        var id = this.model.get('id'),
+            location = id.substring(id.indexOf(':') + 1);
+
+        return {
+            location: location,
+        };
+    },
+
+    ui: _.defaults({
+        chartRegion: '#cuahsi-chart-region',
+        tableRegion: '#cuahsi-table-region',
+    }, ResultDetailsBaseView.prototype.ui),
+
+    regions: {
+        statusRegion: '#cuahsi-status-region',
+        switcherRegion: '#cuahsi-switcher-region',
+        chartRegion: '#cuahsi-chart-region',
+        tableRegion: '#cuahsi-table-region',
+    },
+
+    modelEvents: {
+        'change:mode': 'showChartOrTable',
+    },
+
+    initialize: function() {
+        this.model.set('mode', 'table');
+        // this.model.fetchCuahsiValues();
```

Comment:
Similar to previous, prefer removing unused code, especially that which does not belong (CUAHSI in WQP).